homeassistant/components/vicare/strings.json

```
@@ -65,6 +65,21 @@
         "name": "Heating"
       }
     },
+    "fan": {
+      "ventilation": {
+        "name": "Ventilation",
+        "state_attributes": {
+          "preset_mode": {
+            "state": {
+              "permanent": "permanent",
+              "ventilation": "schedule",
+              "sensorDriven": "sensor",
+              "sensorOverride": "sensorOverride"
```

As hassfest pointed out, these are not valid translations keys.

```suggestion
              "sensorDriven": "sensor",
              "sensorOverride": "sensorOverride"
```

---

The key in this case comes from the api, so I need some kind of in place translation here, right? Is there some helper available to do this?

---

There is no helper for this. You'd need to define a mapping between API and HA in this case.